devops/actions/run-tests/benchmark/action.yml

```diff
@@ -140,6 +140,13 @@ runs:
     shell: bash
     run: |
       git clone -b unify-ci https://github.com/intel/llvm-ci-perf-results
+  # Linux tools installed during docker creation may not match the self-hosted
+  # kernel version, so we need to install the correct version here.
```

Comment:
How exactly would it match the host? Does `uname -r` report host information or container information?

Reply:
Ok I did minimal research https://stackoverflow.com/questions/31012297/uname-a-returning-the-same-in-docker-host-or-any-docker-container


>Docker uses the host operating system kernel, there is no custom or additional kernel inside the container. All containers running on the machine are sharing this "host" kernel.

Reply:
Yep, I tested this locally - kernel reported in docker was exactly like my host.

I pulled sycl image, where linux-tools package was installed (in version A), then run `apt install -y linux-tools-$(uname -r)` in docker and new version B appeared and `perf` started to work.

Also, as I mentioned above - I tested this on BMG machine with other kernel and worked as well (https://github.com/intel/llvm/actions/runs/18881499077/job/53887826008?pr=20493#step:19:726)